tests/page-objects/lending-bar.ts

```diff
@@ -10,4 +10,80 @@ export class LendingBar {
 
     this.iaBookActions = this.page.locator('ia-book-actions');
   }
+
+  async gotoPage(uri: string) {
```

Comment:
i think that this is unnecessary, we can reuse the one existing in `detailsPage`